nephos/composer/install.py

```diff
@@ -31,7 +31,7 @@
     cm_create,
     cm_read,
     ingress_read,
-    secret_create
+    secret_create,
```

Comment:
Good attention to detail, @inzamam-iqbal. It's indeed better to end with comma to improve change detection for `git`, etc.